lib/joken/signer.ex

```diff
@@ -12,7 +12,7 @@ defmodule Joken.Signer do
   @rs_algorithms ["RS256", "RS384", "RS512"]
   @es_algorithms ["ES256", "ES384", "ES512"]
   @ps_algorithms ["PS256", "PS384", "PS512"]
-  @eddsa_algorithms ["Ed25519", "Ed25519ph", "Ed448", "Ed448ph"]
+  @eddsa_algorithms ["Ed25519", "Ed25519ph", "Ed448", "Ed448ph", "EdDSA"]
```

Comment:
`EdDSA` is used as the standard `alg` in the JWT header (not Ed25519)

Reply:
You are absolutely right! 

It is important to keep the Ed25519 as JOSE also accepts that. Thanks for pointing it out :)